ex1/index.js

```diff
@@ -0,0 +1,11 @@
+let n=10;
+
+function numere_pare(n) {
```

Comment:
fisierul asta apare pentru ca (probabil) branchul ex3 (la fel ca ex2) nu a fost creat din branchul "main" dar din branchul ex1.
Se poate vedea in graph

https://github.com/MariusS81/sprint7-Array/network